src/components/ModelSteps/Scan.jsx

```diff
@@ -40,7 +40,7 @@ const Scan = ({ currentStep }) => {
   const { alreadyScan } = useStepperDialog()
 
   const [isFilePickerModalOpen, setIsFilePickerModalOpen] = useState(false)
-  const [fileId, setFileId] = useState('')
+  const [cozyFileId, setCozyFileId] = useState('')
```

Comment:
I don't understand this change. Code is less clear in my opinion

Reply:
Just for clarity :)